packages/fether-react/src/RequireHealthOverlay/RequireHealthOverlay.js

```diff
@@ -82,9 +70,9 @@ class RequireHealthOverlay extends Component {
     return (
       <HealthModal
         fullscreen={fullscreen}
-        healthPercentage={payload && payload.percentage}
-        healthStatus={status}
         loading={loading}
+        payload={payload}
+        status={status}
```

Comment:
You could probably use the withHealth HOC directly in HealthModal, so that there are 2 props less to pass around

Reply:
I've pushed a commit with this change